apps/editors/helpers.py

```diff
@@ -220,6 +221,28 @@ def reviewers_score_bar(context, types=None, addon_type=None):
                                          addon_type=addon_type)))
 
 
+@register.inclusion_tag('editors/includes/files_view.html')
+@jinja2.contextfunction
+def all_distinct_files(context, version):
+    '''
+    Returns [file_ref, [Windows / Mac OS X]] if the same file for
+    Windows and Mac OS X was uploaded.
```

Comment:
Is that what we're returning? I think it's `{<file.hash>: [<file>, 'Windows / Mac OS X']}`

Also, this helper is not returning this, it's providing it in the context. How about:
```
"""Only display a file once even if it's been uploaded for several platforms."""
# hashes_to_file will group files per hash:
# {<file.hash>: [<file>, 'Windows / Mac OS X']}
```